app/Transformers/BillTransformer.php

```diff
@@ -159,47 +159,48 @@ protected function paidData(Bill $bill): array
                 'next_expected_match' => null,
             ];
         }
-        // 2023-07-1 sub one day from the start date to fix a possible bug (see #7704)
-        // 2023-07-18 this particular date is used to search for the last paid date.
-        // 2023-07-18 the cloned $searchDate is used to grab the correct transactions.
         /** @var Carbon $start */
-        $start       = clone $this->parameters->get('start');
-        $searchStart = clone $start;
-        $start->subDay();
```

Comment:
Why did you remove this line in relation to issue #7704?

Reply:
I will spawn a instance to reproduce the issue mentioned in #7704.